_data/master/navbars/usage.yml

```diff
@@ -15,6 +15,8 @@ toc:
   path: /usage/bird-rr-config
 - title: IPv6
   path: /usage/ipv6
+- title: IP Pool Manipulation
```

Comment:
Don't like the term manipulation

Maybe just `IP Pools` or `IP Pool Management`.